src/util/performance.cpp

```diff
@@ -251,6 +254,9 @@ void report() {
         }
     }
 
+    Input::add_callback(SDLK_PAGEUP, KMOD_LCTRL, capture_begin);
+    Input::add_callback(SDLK_PAGEDOWN, KMOD_LCTRL, capture_end);
```

Comment:
I don't have easy access to pageup and pagedown. A normal key would be nice here.